var/spack/repos/builtin/packages/rockstar/package.py

```diff
@@ -0,0 +1,41 @@
+import os
+import shutil
+from spack import *
+
+class Rockstar(Package):
+    """Description"""
+
+    homepage = "https://bitbucket.org/gfcstanford/rockstar"
+    url      = "https://bitbucket.org/gfcstanford/rockstar"
+
+    version('develop', git='https://bitbucket.org/gfcstanford/rockstar.git')
+    version('yt', hg='https://bitbucket.org/MatthewTurk/rockstar')
+
+    variant('hdf5', description='Build rockstar with HDF5 support')
```

Comment:
Can you set a default? It makes it more obvious for people who don't know the default default.